.github/workflows/docker_build.yml

```diff
@@ -97,27 +91,6 @@ jobs:
           username: ${{ secrets.REGISTRY_USERNAME }}
           password: ${{ secrets.REGISTRY_PASSWORD}}
 
-      - name: Set docker image tar name
-        id: setimgver
-        run: |
-          calculatedSha=$(git rev-parse --short ${{ github.sha }})
-          echo "::set-output name=IMAGE_TAR_FULLNAME::${{ inputs.project }}-${{ env.COMPONENT_NAME }}-$calculatedSha.tar"
-
-      - name: If save image tar or push tar to minio
-        id: needminio
-        env:
-          MINIO_ENDPOINT_URL: ${{ secrets.MINIO_ENDPOINT_URL }}
-          MINIO_ACCESS_KEY_ID: ${{ secrets.MINIO_ACCESS_KEY_ID }}
-          MINIO_SECRET_ACCESS_KEY: ${{ secrets.MINIO_SECRET_ACCESS_KEY }}
-        run: |
-          if [[ ${MINIO_ENDPOINT_URL} != "" && ${MINIO_ACCESS_KEY_ID} != "" && ${MINIO_SECRET_ACCESS_KEY} != "" ]];then
-          echo "MINIO_ENABLE=true" >> $GITHUB_ENV
-          echo "MINIO_ENABLE set to true!"
-          else
-          echo "MINIO_ENABLE=false" >> $GITHUB_ENV
-          echo "MINIO_ENABLE set to false!"
-          fi
-
       - name: Load to local registry
```

Comment:
we don't need `Load to local registry` now

Reply:
load to local registry for scanning locally.